core/web/pipeline_job_spec_errors_controller.go

```diff
@@ -19,7 +20,7 @@ type PipelineJobSpecErrorsController struct {
 // Destroy deletes a PipelineJobSpecError record from the database, effectively
 // silencing the error notification
 func (psec *PipelineJobSpecErrorsController) Destroy(c *gin.Context) {
-	jobSpec := job.Job{}
+	jobSpec := job.SpecError{}
```

Comment:
Was this broken or just badly named?

Reply:
It was referring to the wrong struct and as a result forcing the method that receives the ID to be int32 when the actual ID is int64.